design-documents/add-scope-to-cms-pages.md

```diff
@@ -0,0 +1,121 @@
+
+## [HLD] MC-13907: Add StoreView Scope to CMS Pages
+
+HLD for [MC-13907](https://jira.corp.magento.com/browse/MC-13907): Allow CMS Page to Be Edited and Viewed in Admin Based On StoreView
+
+**Current state**
+
+CMS content such as pages and blocks don't support MagentoScope like products and categories do. Merchants need to duplicate pages and blocks and assign them to different storeviews in order to get content localized.
+
+**Problems:**
+
+1. Websites with many languages (EMEA region mostly) need to manage huge number of pages to get the content localized. They maintain certain naming convention for pages to simplify the process of finding the localized duplicates, but making changes for that pages and launching them is still very time consuming and tidies work that slows down time to market for content significantly
```

Comment:
Some scenarios will be actually harder to manage in the new design. For instance, if you have multiple websites, for non default languages user will have to duplicate content manually. In the current UI user will need to create only one page.